### ISSUE TITLE
Error while attempting to upgrade with pip install -U

### ISSUE THREAD

**Issue by raniesantos** [justfoolingaround/animdl#150]
"I have upgraded the project to its latest version, checked the open issues for any duplicates, am not spamming multiple same or similar issues and **have not edited the code myself**. In any of those cases, a maintainer may **CLOSE** or **DELETE** the issue without any notices."

From maintainer: Your title should answer at least 2 or 3 of 5W1H questions for fastest responses.

**Describe the bug**
Error when updating through pip. Currently on version 1.5.81 unable to upgrade to 1.5.83 due to issues.

**Full error output**

```
$ pip install -U animdl

Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: animdl in /home/shindo/.local/lib/python3.10/site-packages (1.5.81)
Collecting animdl
  Using cached animdl-1.5.83.tar.gz (76 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-av_ov9mw/animdl_9ebb199821af4cc195e022553a07b689/setup.py'"'"'; __file__='"'"'/tmp/pip-install-av_ov9mw/animdl_9ebb199821af4cc195e022553a07b689/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-bkbsppkb
       cwd: /tmp/pip-install-av_ov9mw/animdl_9ebb199821af4cc195e022553a07b689/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-av_ov9mw/animdl_9ebb199821af4cc195e022553a07b689/setup.py", line 4, in <module>
      with open("requirements.txt") as requirements_txt:
  FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/96/5c/3fe1f41fb07670c68d5b795a6cf16793a7db036c58cecabb2ff5c883cbea/animdl-1.5.83.tar.gz#sha256=c9da7da36233683108f2341a3622ff565996fbcde6c145e59f387e55a5426d52 (from https://pypi.org/simple/animdl/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  Using cached animdl-1.5.82.tar.gz (76 kB)
  Preparing metadata (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-av_ov9mw/animdl_60671e06d6c0474c920acca040e0284c/setup.py'"'"'; __file__='"'"'/tmp/pip-install-av_ov9mw/animdl_60671e06d6c0474c920acca040e0284c/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-vxilt13m
       cwd: /tmp/pip-install-av_ov9mw/animdl_60671e06d6c0474c920acca040e0284c/
  Complete output (5 lines):
  Traceback (most recent call last):
    File "<string>", line 1, in <module>
    File "/tmp/pip-install-av_ov9mw/animdl_60671e06d6c0474c920acca040e0284c/setup.py", line 4, in <module>
      with open("./requirements.txt") as requirements_txt:
  FileNotFoundError: [Errno 2] No such file or directory: './requirements.txt'
  ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/df/23/7afaf1dcd4a5c77af6c41aa458edcaeb4b4d73327b08130bc2e7b60ed384/animdl-1.5.82.tar.gz#sha256=da50f66ecf006f925e73455e65b449fd21348ee23caae64077cf354365819b5b (from https://pypi.org/simple/animdl/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Requirement already satisfied: pycryptodomex==3.14.1 in /usr/lib64/python3.10/site-packages (from animdl) (3.14.1)
Requirement already satisfied: comtypes==1.1.11 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (1.1.11)
Requirement already satisfied: tqdm==4.62.3 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (4.62.3)
Requirement already satisfied: regex==2022.3.15 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (2022.3.15)
Requirement already satisfied: cssselect==1.1.0 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (1.1.0)
Requirement already satisfied: pyyaml==6.0 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (6.0)
Requirement already satisfied: httpx==0.23.0 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (0.23.0)
Requirement already satisfied: anitopy==2.1.0 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (2.1.0)
Requirement already satisfied: click==8.0.4 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (8.0.4)
Requirement already satisfied: lxml==4.8.0 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (4.8.0)
Requirement already satisfied: yarl==1.7.2 in /home/shindo/.local/lib/python3.10/site-packages (from animdl) (1.7.2)
Requirement already satisfied: certifi in /usr/lib/python3.10/site-packages (from httpx==0.23.0->animdl) (2021.10.8)
Requirement already satisfied: sniffio in /home/shindo/.local/lib/python3.10/site-packages (from httpx==0.23.0->animdl) (1.2.0)
Requirement already satisfied: rfc3986[idna2008]<2,>=1.3 in /home/shindo/.local/lib/python3.10/site-packages (from httpx==0.23.0->animdl) (1.5.0)
Requirement already satisfied: httpcore<0.16.0,>=0.15.0 in /home/shindo/.local/lib/python3.10/site-packages (from httpx==0.23.0->animdl) (0.15.0)
Requirement already satisfied: idna>=2.0 in /usr/lib/python3.10/site-packages (from yarl==1.7.2->animdl) (3.3)
Requirement already satisfied: multidict>=4.0 in /home/shindo/.local/lib/python3.10/site-packages (from yarl==1.7.2->animdl) (6.0.2)
Requirement already satisfied: h11<0.13,>=0.11 in /home/shindo/.local/lib/python3.10/site-packages (from httpcore<0.16.0,>=0.15.0->httpx==0.23.0->animdl) (0.12.0)
Requirement already satisfied: anyio==3.* in /home/shindo/.local/lib/python3.10/site-packages (from httpcore<0.16.0,>=0.15.0->httpx==0.23.0->animdl) (3.5.0)
```

**Did the error occur in between an active task (`download` / `stream` / `grab`)?**

No.

**Screenshots [Optional]**

N/A

**Additional information**

- animdl version: 1.5.81 (trying to upgrade to 1.5.83)
- The site and provider this issue is related to: N/A

**Comment by raniesantos** [comment]
It's not a site error, I don't remember how to change the issue label.

**Comment by justfoolingaround** [comment]
Weird, for now please just resort to pip+git installation.

```sh
$ pip install git+https://github.com/justfoolingaround/animdl
```

**Comment by justfoolingaround** [comment]
Fixed as of `1.5.84`.

Thank you for reporting this issue!

**Comment by raniesantos** [comment]
I updated my Fedora system and then I ran:

`pip install --force-reinstall --no-cache-dir -U animdl`

Everything is good now.
